pages/[lng]/categories/index.tsx

```diff
@@ -0,0 +1,151 @@
+/* library package */
+import { FC, useState } from 'react'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
+import { useRouter } from 'next/router'
+import { RiQuestionFill } from 'react-icons/ri'
+import { ProductListByCategory, useI18n } from '@sirclo/nexus'
+/* library template */
+import { useBrand } from 'lib/useBrand'
+import useInfiniteScroll from 'lib/useInfiniteScroll'
+import useWindowSize from 'lib/useWindowSize'
+/* components */
+import SEO from 'components/SEO'
+import Layout from 'components/Layout/Layout'
+import Breadcrumb from 'components/Breadcrumb/Breadcrumb'
+import EmptyComponent from 'components/EmptyComponent/EmptyComponent'
+/* styles */
+import styles from 'public/scss/pages/Categories.module.scss'
+import styleProducts from 'public/scss/pages/Products.module.scss'
+import styleButton from 'public/scss/components/Button.module.scss'
+import styleProduct from 'public/scss/components/Product.module.scss'
+import Loader from 'components/Loader/Loader'
+
+const classesProductCategory = {
+  productContainerClassName: `col-6 product_list ${styles.productsItemContainer}`,
+  stickerContainerClassName: `${styleProduct.product_sticker} ${styleProduct.product_stickerGrid}`,
+  outOfStockLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__outofstock}`,
+  saleLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__sale}`,
+  comingSoonLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__comingsoon}`,
+  openOrderLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__openorder}`,
+  preOrderLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__preorder}`,
+  newLabelClassName: `${styleProduct.product_stickerLabel} ${styleProduct.product_stickerLabel__new}`,
+  productImageContainerClassName: styleProduct.product_link,
+  productImageClassName: styleProduct.product_itemContainerImage,
+  productLabelContainerClassName: styleProduct.product_label,
+  productTitleClassName: styleProduct.product_label__title,
+  productPriceClassName: styleProduct.product_labelPrice,
```

Comment:
```suggestion
  productImageContainerClassName: styleProduct.productImage,
  productImageClassName: styleProduct.productImage,
  productLabelContainerClassName: styleProduct.productLabelContainer,
  productTitleClassName: styleProduct.productTitle,
  productPriceClassName: styleProduct.productPrice,
```

Namain className-nya samain kaya className dari nexus-nya aja biar lebih gampang.

Reply:
kalo class ini saya ngambilnya dari scss component product mang, jadi kalo diubah kayanya perlu ngubah di halaman lain juga, gapapa mang? atau saya bikin class baru lagi aja khusus untuk style Product di page ini?

Reply:
ooh kalo udah kepalang yaudah gpp